.openshift-ci/jobs/integration-tests/Dockerfile

```diff
@@ -23,3 +23,7 @@ RUN mkdir -p "$GOCACHE" && \
 # to work around this, we have to create the directory
 # first.
 RUN mkdir /.ssh && chmod g+rwx /.ssh
+
+# Install hub-comment for commenting on PRs
+RUN wget --quiet https://github.com/joshdk/hub-comment/releases/download/0.1.0-rc6/hub-comment_linux_amd64 && \
+    install hub-comment_linux_amd64 /usr/bin/hub-comment
```

Comment:
This image is built on top of `quay.io/stackrox-io/apollo-ci:collector`, that image should already have `hub-comment` installed in it: https://github.com/stackrox/rox-ci-image/blob/60cb33a19eb42b7f425eb257dada42096dd75d5c/images/collector.Dockerfile#L57-L59

Reply:
That's what I get for just copying from scripts and not checking the base image! I'll remove this